src/libs/algolia/fetchAlgolia/fetchMultipleOffers/fetchMultipleOffers.test.ts

```diff
@@ -0,0 +1,120 @@
+import { PlaylistOffersParams } from 'features/home/types'
+import * as captureAlgoliaError from 'libs/algolia/fetchAlgolia/AlgoliaError'
+import { buildOfferSearchParameters } from 'libs/algolia/fetchAlgolia/buildAlgoliaParameters/buildOfferSearchParameters'
+import { offerAttributesToRetrieve } from 'libs/algolia/fetchAlgolia/buildAlgoliaParameters/offerAttributesToRetrieve'
+import { fetchMultipleOffers } from 'libs/algolia/fetchAlgolia/fetchMultipleOffers/fetchMultipleOffers'
+import * as multipleQueries from 'libs/algolia/fetchAlgolia/multipleQueries'
+import { buildHitsPerPage } from 'libs/algolia/fetchAlgolia/utils'
+import { LocationMode } from 'libs/algolia/types'
+import { env } from 'libs/environment/env'
+
+const mockMultipleQueries = jest.spyOn(multipleQueries, 'multipleQueries')
+const mockCaptureAlgoliaError = jest.spyOn(captureAlgoliaError, 'captureAlgoliaError')
+
+const mockParams1: PlaylistOffersParams = {
+  indexName: 'custom_index',
+  offerParams: {
+    beginningDatetime: undefined,
+    date: null,
+    endingDatetime: undefined,
+    hitsPerPage: 10,
+    isDigital: false,
+    offerCategories: [],
+    offerIsDuo: false,
+    offerIsFree: false,
+    offerSubcategories: [],
+    priceRange: [0, 300],
+    query: 'test',
+    tags: [],
+    timeRange: null,
+    venue: {
+      _geoloc: { lat: 48.8536, lng: 2.34199 },
+      info: 'PARIS 6',
+      label: 'Cinéma St André des Arts',
+      venueId: 26235,
+    },
+  },
+  locationParams: {
+    aroundMeRadius: 100,
+    aroundPlaceRadius: 100,
+    selectedLocationMode: LocationMode.AROUND_ME,
+    userLocation: { latitude: 48.90374, longitude: 2.48171 },
+  },
+}
+
+const mockParams2: PlaylistOffersParams = {
+  offerParams: {
+    beginningDatetime: undefined,
+    date: null,
+    endingDatetime: undefined,
+    hitsPerPage: 5,
+    isDigital: false,
+    offerCategories: [],
+    offerIsDuo: false,
+    offerIsFree: false,
+    offerSubcategories: [],
+    priceRange: [0, 300],
+    query: 'another',
+    tags: [],
+    timeRange: null,
+    venue: {
+      _geoloc: { lat: 48.8536, lng: 2.34199 },
+      info: 'PARIS 6',
+      label: 'Cinéma St André des Arts',
+      venueId: 26235,
+    },
+  },
+  locationParams: {
+    aroundMeRadius: 100,
+    aroundPlaceRadius: 100,
+    selectedLocationMode: LocationMode.AROUND_ME,
+    userLocation: { latitude: 48.90374, longitude: 2.48171 },
+  },
+}
```

Comment:
```suggestion
const mockParams2: PlaylistOffersParams = { ...mockParams1, indexName: 'customIndex' }
```
Et déclarer `mockParams1` sans l'`indexName`.
Je trouve qu'on y gagne en lisibilité, on n'a pas a cherché les différences entre les deux objets.